bin/setup-services.sh

```diff
@@ -7,8 +7,8 @@
 set -eo pipefail
 
 # Set up S3
-python bin/s3.py delete "http://minio:9000/testbucket"
-python bin/s3.py create "http://minio:9000/testbucket"
+python bin/s3.py delete "http://minio:9000/publicbucket"
+python bin/s3.py create "http://minio:9000/publicbucket"
```

Comment:
Previously, we used "testbucket", but it was unclear whether that should have been a public bucket or a private bucket and (in fact) it was set up as a private bucket (unhelpful). In changing the local dev and test environments to use a public bucket as the first bucket listed in SYMBOL_URLS, I decided to make it clearer that it was public by renaming it "publicbucket".